Add links to README

infominer33 opened the pull request:
As you see, there were a few files in the directory that didn't have links.

If there is no reason for them to be separate from the others, I can remove the divider.

If there is a reason they were left out from the list originally, other than oversight, I would be interested to know. 

If there is a good reason why they aren't on the list, then I won't bother sending pull-requests for the other repositories that I found w a similar situation.

infominer33 committed:
Update README.md

infominer33 committed:
remove extra space

infominer33 committed:
Add links for files in directory not already listed in README

infominer33 committed:
uniformity

ChristopherA reviewed (approved):
I believe that these were all added late, but are acceptable.

infominer33 committed:
remove spaces and references to them

infominer33 commented:
In the most recent commit, I've removed all spaces from filenames, and changed links as appropriate in the README. 

For more details, see: [rwot1-sf/PR#92](https://github.com/WebOfTrustInfo/rwot1-sf/pull/92)

infominer33 committed:
replace spaces in filenames w -

infominer33 committed:
change to h2
